new/LIT7276MarkOne.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7276MarkOne" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Mark 1:23-27</title>
+                <title xml:lang="en" xml:id="t2">Prayer against devils and for the healing of the man with an unclean spirit</title>
+                <title xml:lang="gez" xml:id="t3">ወሀሎ፡ ፩ብእሲ፡ ዘአኃዞ፡ ጋኔን፡ በምኵራብ፡</title>
+                <title xml:lang="gez" corresp="#t3" type="normalized">Wa-hallo 1 bǝʾǝsi za-ʾaḫāzo gānen ba-mǝkʷǝrāb</title>
+                <title xml:lang="en" corresp="#t3">There was a man in the synagogue, who was caught by a demon</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Part from the <ref type="work" corresp="LIT2711Mark">Gospel of Mark</ref> serving as a magic prayer against demons
+                    and for the healing from unclean spirits.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Missal"/>
```

Comment:
I don't think "Missal" fits here, as the text is not an instruction for how to celebrate the Eucharist.